File: Modul-1/Soal 5/PRAK105-2410817210014-ZAVIER GHANI.py
def cetak_pesan():
    a = 9
    b = 5
    x = 8
    y = 8

    sisa_a_b = a % b
    sisa_x_y = x % y
    total_sisa = sisa_a_b + sisa_x_y

    print("Variabel a bernilai:", a)
    print("Variabel b bernilai:", b)
    print("Variabel x bernilai:", x)
    print("Variabel y bernilai:", y)
    print("Total sisa bagi dari a dibagi b dan x dibagi y adalah:", total_sisa)

cetak_pesan()

File: Modul-1/Soal 5/PRAK105-2410817210014-ZAVIERGHANI.c
#include <stdio.h>

int main() {
    int a = 9;
    int b = 5;
    int x = 8;
    int y = 8;

    int sisa_a_b = a % b;
    int sisa_x_y = x % y;
    int total_sisa = sisa_a_b + sisa_x_y;

    printf("Variabel a bernilai: %d\n\n", a);
    printf("Variabel b bernilai: %d\n\n", b);
    printf("Variabel x bernilai: %d\n\n", x);
    printf("Variabel y bernilai: %d\n\n", y);
    printf("Total sisa bagi dari a dibagi b dan x dibagi y adalah: %d\n", total_sisa);

    return 0;
}